azure-pipelines.yml

```diff
@@ -128,6 +128,13 @@ jobs:
     displayName: 'Check installed packages'
   - powershell: |
       cd testsuite
+      $env:OPENBLAS_NUM_THREADS=1
+      $env:GOTO_NUM_THREADS=1
+      $env:OMP_NUM_THREADS=1
+      $env:MKL_NUM_THREADS=1
+      # limit to 2 workers to avoid overloading the CI
+      $env:PYTEST_XDIST_AUTO_NUM_WORKERS=1
+
       pytest MDAnalysisTests --disable-pytest-warnings -n auto --timeout=200 -rsx --cov=MDAnalysis
```

Comment:
Here we already have `-n auto` – how does this interact with `PYTEST_XDIST_AUTO_NUM_WORKERS`?

Reply:
pytest will read `PYTEST_XDIST_AUTO_NUM_WORKERS` when `-n auto`. It's equivalent to `pytest -n $PYTEST_XDIST_AUTO_NUM_WORKERS`